src/main/java/uk/gov/companieshouse/api/testdata/model/rest/Jurisdiction.java

```diff
@@ -1,12 +1,42 @@
 package uk.gov.companieshouse.api.testdata.model.rest;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.api.testdata.model.entity.Address;
 
 public enum Jurisdiction {
+
     @JsonProperty("england/wales")
-    ENGLAND_WALES, 
+    ENGLAND_WALES(new Address(
+            "Companies House",
+            "Crownway",
+            "United Kingdom",
+            "Cardiff",
+            "CF14 3UZ"
+    ), "Wales"),
     @JsonProperty("scotland")
-    SCOTLAND, 
+    SCOTLAND(new Address(
+            "4th Floor Edinburgh Quay 2",
+            "139 Fountain Bridge",
+            "United Kingdom",
+            "Edinburgh",
+            "EH3 9FF"
+    ), "Scotland"),
     @JsonProperty("northern-ireland")
-    NI;
+    NI(new Address(), "NI");
+
+    private Address address;
+    private String countryOfResidence;
```

Comment:
I'm not convinced of storing these in the enum. It would bring logic to this rest model and also couple both entity and rest model?

It's not as if the address and the country of residence are logically assigned to a jurisdiction but only linked for our current requirements. Maybe we could do something like this for the company prefix?